README.md

```diff
@@ -217,13 +217,19 @@ GitHub Actions allows users to select a specific version of the action to use,
 based on release tags. This script simplifies this process by performing the
 following steps:
 
-1. **Retrieving the latest release tag:** The script starts by fetching the most
-   recent release tag by looking at the local data available in your repository.
-1. **Prompting for a new release tag:** The user is then prompted to enter a new
-   release tag. To assist with this, the script displays the latest release tag
-   and provides a regular expression to validate the format of the new tag.
-1. **Tagging the new release:** Once a valid new tag is entered, the script tags
-   the new release.
-1. **Pushing the new tag to the remote:** Finally, the script pushes the new tag
-   to the remote repository. From here, you will need to create a new release in
-   GitHub and users can easily reference the new tag in their workflows.
+1. **Retrieving the latest release tag:** The script starts by fetching the
+   most recent semver release tag of the current branch, by looking at the
+   local data available in your repository.
+1. **Prompting for a new release tag:** The user is then prompted to enter a
+   new release tag. To assist with this, the script displays the tag retrieved
+   in the previous step, and validates the format of the inputted tag (vX.X.X).
+   The user is also reminded to update the version field in package.json.
+1. **Tagging the new release:** The script then tags a new release and syncs
+   the separate major tag (e.g. v1, v2) with the new release tag
+   (e.g. v1.0.0, v2.1.2). When the user is creating a new major release, the
+   script auto-detects this and creates a `releases/v#` branch for the previous
+   major version.
```

Comment:
What would be the reason for creating a separate `releases/v#` branch? Wouldn't the release tag already cover this?

Reply:
I was thinking, to allow patch fixes for a previous release.

Reply:
Sorry for the delay. That makes sense to me!